.travis.yml

```diff
@@ -1,32 +1,42 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
+# Not using self-jobs-range as we need to exclude PGSQL which will have broken phpunit tests
```

Comment:
@emteknetnz Is there a specific module that is failing against PostgreSQL that we could reference here to resolve in the future?

Reply:
This is for CWP so we don't support PGSQL

In the `cwp` provision in the jobs matrix there's no PGSQL.  I used the `self` provision which is for recipes. We didn't bother to make a separate `cwp-self` because there's so few of them

Reply:
Makes sense, that's just a slightly different explanation to what you've documented here - want to tweak it?

Reply:
done